src/vma/sock/sock-redirect.cpp

```diff
@@ -1664,10 +1537,6 @@ int select(int __nfds,
 	   fd_set * __exceptfds,
 	   struct timeval * __timeout)
 {
-	BULLSEYE_EXCLUDE_BLOCK_START
-	if (!orig_os_api.select)	get_orig_funcs();
-	BULLSEYE_EXCLUDE_BLOCK_END
-
 	if (!g_p_fd_collection)
```

Comment:
it it necessary to check **!g_p_fd_collection** ?
we check it during _select()_, _pselect()_, _poll()_, _ppoll()_ but not from _epoll_wait()_.